src/klr.core45/tpa.cpp

```diff
@@ -6,78 +6,82 @@
 
 BOOL CreateTpaBase(LPWSTR** ppNames, size_t* pcNames, bool bNative)
 {
-    const size_t count = 32;
+    const size_t count = 34;
```

Comment:
@ChengTian Yes, you can check this file separately. The dpa running in the second round of the build and find these new dependencies for klr.core45.managed. It may be introduced by you but by CoreCLR.